dynawo/sources/Modeler/Common/DYNDynamicData.cpp

```diff
@@ -303,13 +301,9 @@ DynamicData::getParametersSet(const string& modelId, const string& parFile, cons
   // Parameters file not already loaded
   parameters::XmlImporter parametersImporter;
 
-  try {
-    shared_ptr<ParametersSetCollection> parametersSetCollection = parametersImporter.importFromFile(canonical(parFile, rootDirectory_));
-    referenceParameters_[parFile] = parametersSetCollection;
-    return parametersSetCollection->getParametersSet(parId);
-  } catch (const xml::sax::parser::ParserException& exp) {
-    throw DYNError(Error::MODELER, XmlFileParsingError, parFile, exp.what());
-  }
+  shared_ptr<ParametersSetCollection> parametersSetCollection = parametersImporter.importFromFile(canonical(parFile, rootDirectory_));
```

Comment:
Please keep the proper handle of this exception

Reply:
The exception is already handled in the PAR API and will never be caught here